includes/class-wcpdf-frontend.php

```diff
@@ -103,27 +104,42 @@ public function woocommerce_api_invoice_number ( $data, $order ) {
 	 * Download invoice frontend shortcode
 	 */
 	public function download_invoice_shortcode( $atts ) {
+		return $this->generate_document_link_shortcode( $atts );
+	}
+
+	/**
+	 * Download pdf frontend general shortcode
+	 */
+	public function download_pdf_shortcode( $atts ) {
```

Comment:
Maybe we don't need this 2 functions and we could use the `generate_document_link_shortcode()` directly?

```php
add_shortcode( 'wcpdf_download_pdf', array( $this, 'generate_document_link_shortcode' ) );
```